src/components/editions/headerImage.tsx

```diff
@@ -113,7 +113,10 @@ const sizes: Sizes = {
 };
 
 const fullWidthSizes: Sizes = {
-	mediaQueries: [],
+	mediaQueries: [
+		{ breakpoint: 'tablet', size: '100vw' },
+		{ breakpoint: 'wide', size: '100vw' },
+	],
 	default: '100vw',
```

Comment:
If these are all `100vw` do you need anything beyond the `default`?

Reply:
It was my suggestion to add breakpoint values but had envisaged as `px`s - if 100vw suffices then, no, there's no need for anything more than the default.

Reply:
Yep, the image dimensions are derived from `size` so px fixes the width which we don't want here. Will revert the change!